### ISSUE TITLE
allow to set NATS maximum payload

### ISSUE THREAD

**Issue by danielchg** [konstellation-io/kre#377]
Have been detected the below logs in a Runtime Entrypoint after be running for a while:
```
Exception on gRPC call : nats: Maximum Payload ExceededTraceback (most recent call last):  File "/app/src/kre_grpc.py", line 42, in process_message    timeout=self.config.request_timeout)  File "/usr/local/lib/python3.7/dist-packages/nats/aio/client.py", line 902, in request    await self.publish_request(subject, inbox.decode(), payload)  File "/usr/local/lib/python3.7/dist-packages/nats/aio/client.py", line 626, in publish_request    raise ErrMaxPayloadnats.aio.errors.ErrMaxPayload: nats: Maximum Payload Exceeded
```
Following are the logs of the NATS Server when starting:

```
[1] 2020/10/14 07:16:46.146051 [INF] STREAM: Server is active
[1] 2020/10/14 07:16:46.146075 [INF] STREAM: Recovering the state...
[1] 2020/10/14 07:16:46.146177 [INF] STREAM: No recovered state
[1] 2020/10/14 07:16:46.146568 [INF] STREAM: Message store is FILE
[1] 2020/10/14 07:16:46.146575 [INF] STREAM: Store location: /nats-datastore
[1] 2020/10/14 07:16:46.146637 [INF] STREAM: ---------- Store Limits ----------
[1] 2020/10/14 07:16:46.146640 [INF] STREAM: Channels:                  100 *
[1] 2020/10/14 07:16:46.146643 [INF] STREAM: --------- Channels Limits --------
[1] 2020/10/14 07:16:46.146646 [INF] STREAM:   Subscriptions:          1000 *
[1] 2020/10/14 07:16:46.146649 [INF] STREAM:   Messages     :       1000000 *
[1] 2020/10/14 07:16:46.146652 [INF] STREAM:   Bytes        :     976.56 MB *
[1] 2020/10/14 07:16:46.146658 [INF] STREAM:   Age          :     unlimited *
[1] 2020/10/14 07:16:46.146661 [INF] STREAM:   Inactivity   :     unlimited *
[1] 2020/10/14 07:16:46.146664 [INF] STREAM: ----------------------------------
[1] 2020/10/14 07:16:46.146667 [INF] STREAM: Streaming Server is ready
```

**Comment by xiscocapllonch** [comment]
Maximum Payload (`max_payload`) is configurable in NATS core?
https://docs.nats.io/nats-server/configuration#limits

**Comment by xiscocapllonch** [comment]
It's confirmed that the issue occurs when we send a python List that converted to `json` has more than 1MB, 1.3MB.

**Comment by jonasdacruz** [comment]
This will be covered within the issue #381

**Comment by guumaster** [comment]
solved with message compression #405
